tests/components/homekit_controller/specific_devices/test_vocolinc_flowerbud.py

```diff
@@ -0,0 +1,120 @@
+"""
+Test against characteristics captured from a VOCOLinc Flowerbud.
+
+https://github.com/home-assistant/core/issues/26180
+"""
+from aiohomekit.model.characteristics import CharacteristicsTypes
+from aiohomekit.model.services import ServicesTypes
+
+from homeassistant.components.humidifier import DOMAIN
+
+from tests.components.homekit_controller.common import setup_test_component
+
+ACTIVE = ("humidifier-dehumidifier", "active")
+CURRENT_HUMIDIFIER_DEHUMIDIFIER_STATE = (
+    "humidifier-dehumidifier",
+    "humidifier-dehumidifier.state.current",
+)
+TARGET_HUMIDIFIER_DEHUMIDIFIER_STATE = (
+    "humidifier-dehumidifier",
+    "humidifier-dehumidifier.state.target",
+)
+RELATIVE_HUMIDITY_CURRENT = ("humidifier-dehumidifier", "relative-humidity.current")
+
+VOCOLINC_HUMIDIFIER_SPRAY_LEVEL = (
+    "humidifier-dehumidifier",
+    f"Unknown Characteristic {CharacteristicsTypes.Vendor.VOCOLINC_HUMIDIFIER_SPRAY_LEVEL}",
+)
+
+
+def create_diffuser_service(accessory):
```

Comment:
The tests in `specific_devices` are really about loading a device fixture from JSON in `tests/fixtures/homekit_controller`that have been captured from real devices (i usually just use whats in `.storage/homekit_controller-entity-map`). I mostly use them to check that things have been enumerated correctly. There are some tests for device specific regressions, but mostly the rule is if you are mocking characteristics it belong in `test_humidifier.py`, if its using data from a real device it goes in `specific_devices`.

Reply:
I just went ahead and removed it, since it seemed redundant. Thank you for explaining the purpose of those tests.